src/content/certifications/exam-faqs/terraform-authoring-operations-professional.mdx

```diff
@@ -0,0 +1,88 @@
+<!-- Each "## Heading Two" section in this document will be used to populate an FAQ item -->
+
+## Exam Availability
+
+Coming soon! 
+
+## Prerequisites
+
+We strongly recommend passing the associate-level Terraform exam before taking the professional-level exam. Practitioners who have extensive experience with Terraform in a production environment, and understand the concepts covered in the associate exam, may be able to successfully pass the professional-level exam.
+
+- HashiCorp Certified: Terraform Associate Certification (recommended)
+- Linux skills such as list and edit files via command terminal
```

Comment:
```suggestion
- Linux skills such as the ability to list and edit files via command terminal
```

Do we know what editors are available to the reader to edit files via the command line? Do we intended that they'll be using vim/emacs/nano/etc? Or will they have access to a full text editor such as VS Code?